website/data/docs-nav-data.json

```diff
@@ -0,0 +1,1224 @@
+[
```

Comment:
As part of this work, I've converted `navigation.js` files to `nav-data.json` files per [MKTG-032](https://docs.google.com/document/u/2/d/1FPyWid2NmNI9E_6lH2UIMuEPqjqDCGe3F8wraPRQHQ8).